ansible/playbook/roles/generate_inventory/templates/cloud.inventory.j2

```diff
@@ -50,17 +54,18 @@ openshift_master_htpasswd_users={'admin': '$apr1$DloeoaY3$nqbN9fQBkyXgbj58buqEM.
 
 
 {% set host_info = 'localhost ansible_connection=local' if ((use_local is defined) and (use_local | bool)) else ip_address %}
+{% set private_ip_address = private_ip_address if (private_ip_address is defined) else ip_address %}
 
 # host group for masters
 [masters]
-{{ host_info }} openshift_public_hostname={{ ip_address }} openshift_ip={{ ip_address }}
+{{ host_info }} openshift_public_hostname={{ ip_address }} openshift_hostname={{ private_ip_address }}
```

Comment:
According to my issue ticket, for local or hetzner deployment, then it is required to specify `openshift_ip` and for openstack `openshift_hostname`. Are you sure ? Have you tested ?

Reply:
I thought you had tested that setup... So I must have miss-understood what exactly you tested.

Reply:
My ticket is clear. See text's comment and screenshot

Reply:
I will test this PR with the local VM as well

Reply:
Yes, my bad :)

Reply:
I am testing the approach with the local VM. If it works, then I would much rather gown down the root of this PR, since it avoid having complex logic of when to activate/deactivate fields